projects/igniteui-angular/src/lib/slider/slider.component.spec.ts

```diff
@@ -622,6 +623,149 @@ describe('IgxSlider', () => {
         expect(slider.minValue).toEqual(fix.componentInstance.minValue);
         expect(slider.maxValue).toEqual(fix.componentInstance.maxValue);
     });
+
+    it('should track min/maxValue if lower/upperBound are undefined (issue #920)', () => {
+        const fixture = TestBed.createComponent(SliderTestComponent);
+        fixture.detectChanges();
+
+        const slider = fixture.componentInstance.slider;
+
+        fixture.componentInstance.slider.value = 5;
```

Comment:
You should not set the value in this test. You're doing this in the next test. This test is meant to test only the tracking of the lower and upper bounds. Test only the functionality described in the test title.